src/Components/Landing/Landing.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import "./landing.css";
+
+const Landing = () => {
+  return (
+    <section id="landing">
+      <div className="container">
+        <div className="heading">
+          <h1 className="title">Listening is everything.</h1>
+        </div>
+        <div className="para">
+          Listen to millions of music, without interruptions.
```

Comment:
Can we make this something like "millions of hours of music" or "millions of songs"?

Reply:
On it.